docs/faq.md

```diff
@@ -86,6 +86,7 @@ WebTorrent is still pretty new, but it's already being used in cool ways:
 - **[PeerFast][peerfast]** - First P2P Internet Speed Test ([source code][peerfast-source])
 - **[TorrentMedia][torrentmedia]** - Fully-featured desktop WebTorrent client
 - **[Gaia 3D Star Map][gaia]** - 2 million stars, rendered in 3D with WebGL, WebVR, and WebTorrent
+- **[Watchtor](https://watchtor.herokuapp.com)** - A minimalistic approach for online torrent watching
```

Comment:
Oops, my bad.
I was really sleepy last night and missed the common link formatting we're using there.
Fixed now.

Thanks @feross!